### PR TITLE
Update templates.yml civilianpeterparker

### PR DESCRIPTION
**Pull request by SirRicardo**

### CODE REVIEW

**Comment by ItzHarkomal** [comment]
There should be a space before bfc9e44d-4e70-4544-8628-df9ce9be1bfe in cached-skin-uuid

**Comment by ItzHarkomal** [comment]
In line 30, the name should be like this: Civilian Peter Parker

**Comment by ItzHarkomal** [comment]
In line 30 you put Civilian Peter P{arker. Remove the { please.

File: Citizens/templates.yml [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+      cached-skin-uuid: bfc9e44d-4e70-4544-8628-df9ce9be1bfe 
+      player-skin-name: sirricardo      
+      nameplate-visible: false       
+      player-skin-signature: hm99HBZiXR+qxi0tFhzjKUqS/8KK2XFh4JZ9aY6EVON3UuhuD1qlWVd1NtBVaNt8Vv7+L0+XpLD9HXdVKwC5eYZ5dHTPomnG8LF49NEw8Pq9k4NSRIphtWNa2+D5XUveiyqOVvt+clCiJLP4spP4xu9Fu6pRt16IGW3VEq2G5L3ZKyJSbBfiasB3Q2fn4jXGRnVvBxU666EUDthYqmhbnRlDb0GiJ470EuGuS1hoYMpAyZ/wW1M/Y4pVGKsjmiQNkgGje2IdjiknqC3gb7DJEqaqKtq9UtkLF33g5jAr0QotTa1zOlpCwPoy+YvtMdtsQXtlDyQrGArdx6ba46oAs9e3T1vN51EbIZ408xAFOlzP1NVlh1eU4E2TnqPs4ry/McFqIhV36YZlJRiiry3RxlWmlsMdfyNntCuZK/xB2N3SEPhwXWjCZrt4SyUeAyGkK2trdzTDGMLLG9lDJNMWefPFwb6/WGuz6RVDq9sca7xzFjbGbtUjO2QcQgqRmKvkO2ju6NSegbcBdBlCmRX4gy8cgkhSdVXiG4szONk4iJG8H1Jvzz0/yTOyv9tWXKft7w0KOL/uEmGrfA2v2XSnZGxzMeb0nNvoOOMg529FDP6k9qlb0uMMErGqs25AM8w0fKAL2CwRIlDLxWi+CxbExiCGH1pVL7+mZ5e6JY8x76I=
+      avoidwater: false       
+      usedefaultstuckaction: true       
+    uuid: bfc9e44d-4e70-4544-8628-df9ce9be1bfe      
+    traitnames: spawned,type,inventory,owner,gravity,location       
+    name: Civilian Peter Parker
+piccolo:
```
Comment (malonnnn) [inline Citizens/templates.yml]: why is piccolo being added here?

**Pull request closed without merging**